paltas/Utils/cosmology_utils.py

```diff
@@ -109,6 +109,34 @@ def absolute_to_apparent(mag_absolute,z_light,cosmo,
 
 	return mag_apparent
 
+def apparent_to_absolute(mag_apparent,z_light,cosmo,include_k_correction=True):
```

Comment:
You could avoid some duplication here by making one function (maybe starting with an underscore) that outputs the apparent-absolute offset and then call it from both apparent_to_absolute and absolute_to_apparent. That would make it immediately obvious there is not e.g. a minus sign error somewhere.